src/notification/email/EMailNotificationTask.ts

```diff
@@ -181,11 +181,15 @@ export default class EMailNotificationTask implements NotificationTask {
     const optionalComponents = [await EmailComponentManager.getComponent(EmailComponent.MJML_TABLE)];
     let templateName: string;
     if (data.invoiceStatus === 'paid') {
-      data.buttonUrl = data.invoiceDownloadUrl;
+      if (user.role === UserRole.EXTERNAL) {
+        data.buttonUrl = data.evseScanPayInvoiceDownloadURL;
+      } else {
+        data.buttonUrl = data.invoiceDownloadUrl;
+      }
       templateName = 'billing-new-invoice-paid';
     } else {
       data.buttonUrl = data.payInvoiceUrl;
-      templateName = 'billing-new-invoice-unpaid';
+      templateName = 'billing-new-invoice-unpaid-scan-pay';
```

Comment:
Just to confirm that the _billing-new-invoice-unpaid_  template is no longer used is ok ?